src/materials/ConcreteASREigenstrain.C

```diff
@@ -302,7 +302,7 @@ ConcreteASREigenstrain::computeDerivative(unsigned qp, Real scalar)
     {
       f = 1.0 + _alpha * I_sigma / (3.0 * -_f_compress);
       if (f < 1.0)
-        mooseError("Wrong retardation for ASR latency time calculation!");
+        mooseAssert("f < 1.0", "Wrong retardation for ASR latency time calculation!");
```

Comment:
Same as above -- delete line 304 and change the assertion to `"f >= 1.0"`.

Reply:
Will do.